File: Oficinas17/uri1173.cpp
/*
 * Leonardo Deliyannis Constantin
 * URI 1173 - Preenchimento de Vetor I
 */

#include <iostream>
using namespace std;

int main(){
	int i;
	int N[10];
	cin >> N[0];
	cout << "N[0] = " << N[0] << endl;
	for(i = 1; i < 10; i++){
		N[i] = N[i-1] * 2;
		cout << "N[" << i << "] = " << N[i] << endl;
	}
	return 0;
}

File: Oficinas17/uri1173.java
/*
 * Leonardo Deliyannis Constantin
 * URI 1173 - Preenchimento de Vetor I
 */

import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

class Main{
	public static void main(String args[]) throws IOException{
		Locale.setDefault(new Locale("en", "US"));
		Scanner sc = new Scanner(System.in);
		int i;
		int N[] = new int[10];
		N[0] = sc.nextInt();
		System.out.println("N[0] = " + N[0]);
		for(i = 1; i < 10; i++){
			N[i] = N[i-1] * 2;
			System.out.println("N[" + i + "] = " + N[i]);
		}
	}
}